client/locations-list-map.js

```diff
@@ -1701,10 +1696,8 @@ $(() => {
 
     updateFilters(filters);
 
-    if (showMap) {
-      $map.show();
-      loadMapScript(data, filters);
-    }
+    $map.show();
```

Comment:
is this set to `display: none` by default? By removing this query param do we need that anymore?

Reply:
Good call. Inspired by that, I reworked handling of initial show vs. hide of several elements.